practice/batch/services/service_actions_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'rspec'
+require_relative 'services_actions'
+
+
+describe 'Services' do
+
+  before(:each) do
+    @service_name = 'vpnagent'
+    @status_stopped = SERVICES::STOPPED
+    @status_running = SERVICES::RUNNING
+  end
+
+  it 'should be stopped and started again properly' do
+
+    if SERVICES::service_status(@service_name, @status_running)
+      SERVICES::service_stop(@service_name)
+    else
+      SERVICES::service_start(@service_name)
+      SERVICES::service_stop(@service_name)
+    end
+    sleep 1
+    expect(SERVICES::service_status(@service_name, @status_stopped)).to eql true
```

Comment:
After we stopped the service I would like to see it in the next way: expect(service_status).to eql @status_stopped (syntax is approximate)